### ISSUE TITLE
directory problem on Windows

### ISSUE THREAD

**Issue by keesvp** [JuliaGraphics/Tk.jl#62]
running tests.jl
img = Image(Pkg.dir("Tk", "examples", "weather-overcast.gif"))
ERROR: TclError("couldn't open \"C:UsersKees.juliaTkexamplesweather-overcast.gif\": no such file or directory")
 in tcl_eval at C:\Users\Kees.julia\Tk\src\tkwidget.jl:97
 in tcl_eval at C:\Users\Kees.julia\Tk\src\tkwidget.jl:91
 in tcl at C:\Users\Kees.julia\Tk\src\core.jl:48
 in Image at C:\Users\Kees.julia\Tk\src\widgets.jl:321

Pkg.dir returns:
"C:\Users\Kees\.julia\Tk\examples\weather-overcast.gif"
the backslashes disappear.

**Comment by hustf** [comment]
Notice how just some of the backslashes disappear in the original post. I did what @Varanas and @tkelman have probably done and traced this strange behaviour to tkwidget.jl:91 function
````    function tcl_eval(cmd,tclinterp)
code = ccall((:Tcl_Eval,libtcl), Int32, (Ptr{Void}, Ptr{UInt8}),  tclinterp, cmd)````

Using 0.3.11 and the very oldest version of the package, the same problem seems to have been present on Windows since .... 32 bit to 64 bit perhaps?

Placing format checks inside of the image function should work if there is no filesystem where \ may be part of the filename. Perhaps this bad behaviour is an acceptable one, or perhaps additional checks in Image would cover that.

...but can't such unexpected behaviour from a core  function easily lead to erratic behaviour from tk in it other cases too? Is the fault really with tk, or in the called library?

**Comment by aviks** [comment]
Fixed by #147 I guess.
